tuf/builder.go

```diff
@@ -198,7 +199,7 @@ func (rb *repoBuilder) GetLoadedVersion(roleName string) int {
 		}
 	}
 
-	return 0
+	return 1
```

Comment:
nit: it seems like it'd be helpful to make a `MinMetadataVersion` or similar constant since we reuse `1` throughout the code